crates/bevy_ui/Cargo.toml

```diff
@@ -34,3 +34,4 @@ taffy = "0.1.0"
 serde = { version = "1", features = ["derive"] }
 smallvec = { version = "1.6", features = ["union", "const_generics"] }
 bytemuck = { version = "1.5", features = ["derive"] }
+thiserror = "1.0.37"
```

Comment:
What is this and how is it relevant for this PR?

Reply:
I use this for `SliderValueError`, for making error displayable.